### PR TITLE
feat(perf): Add more Requests module documentation

### PR DESCRIPTION
**Pull request by gggritso**
- Add widget note
- Add manual instrumentation notes

### CODE REVIEW

File: docs/product/performance/requests.mdx [hunk 1]
```diff
@@ -21,19 +21,30 @@ On the [**Requests** page](#requests-page), you get an overview of the domains t
...
 </Note>
 
+Availability of HTTP request monitoring depends on the SDK your application uses. In most cases, Sentry's SDKs automatically enable HTTP request tracking. You can check your SDK's <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation documentation</PlatformLink> to see if it includes tracking HTTP requests.
+
+### Span Eligibility
+
+If the SDK you are using doesn't automatically instrument HTTP requests, you can use <PlatformLink to="/performance/instrumentation/custom-instrumentation/">custom instrumentation</PlatformLink>. For best results, follow these guidelines for your custom spans:
+
+- the `op` field is set to `"http.client"`
+- the `description` field contains the full URL of the request (e.g., `"GET http://my.service.io/some-data"`)
```
[the remark this answers is not in the document]
Comment (gggritso) [reply docs/product/performance/requests.mdx]: Good catch, added!

File: docs/product/performance/requests.mdx [hunk 2]
```diff
@@ -21,19 +21,30 @@ On the [**Requests** page](#requests-page), you get an overview of the domains t
   ></iframe>
 </div>
 
-## Eligibility
-
-Availability of HTTP request monitoring depends on the SDK your application uses. In most cases, Sentry's SDKs automatically enable HTTP request tracking. You can check your SDK's <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation documentation</PlatformLink> to see if it includes tracking HTTP requests.
+## Prerequisites
 
 <Note>
 Network requests using other protocols (FTP, WebSocket, etc.) are not supported at this time.
```
Comment (vivianyentran) [inline docs/product/performance/requests.mdx:27]:
```suggestion
Network requests using non-HTTP protocols (FTP, WebSocket, etc.) are not supported at this time.
```

File: docs/product/performance/requests.mdx [hunk 3]
```diff
@@ -21,19 +21,30 @@ On the [**Requests** page](#requests-page), you get an overview of the domains t
...
+Availability of HTTP request monitoring depends on the SDK your application uses. In most cases, Sentry's SDKs automatically enable HTTP request tracking. You can check your SDK's <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation documentation</PlatformLink> to see if it includes tracking HTTP requests.
+
+### Span Eligibility
+
+If the SDK you are using doesn't automatically instrument HTTP requests, you can use <PlatformLink to="/performance/instrumentation/custom-instrumentation/">custom instrumentation</PlatformLink>. For best results, follow these guidelines for your custom spans:
+
+- the `op` field is set to `"http.client"`
+- the `description` field contains the HTTP method and the full URL of the request (e.g., `"GET http://my.service.io/some-data"`)
+- if the request is to a relative URL (e.g., `"GET /data.json"`) the `server.address` span data value is set to the server address (e.g., `"my.service.io"`)
+- the `http.response.status_code` contains the HTTP response status code (e.g., `"200"`)
```
Comment (vivianyentran) [inline docs/product/performance/requests.mdx:39]:
```suggestion
- The `op` field is set to `"http.client"`.
- The `description` field contains the HTTP method and the full URL of the request (e.g., `"GET http://my.service.io/some-data"`).
- If the request is to a relative URL (e.g., `"GET /data.json"`), the `server.address` span data value should be set to the server address (e.g., `"my.service.io"`).
- The `http.response.status_code` contains the HTTP response status code (e.g., `"200"`).
```

**Review by vivianyentran — APPROVED** [review]
Looks good, thanks for updating!

**Pull request merged**